mrp_bom_produce_delay/view/mrp_bom.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+Copyright (C) 2024 - Today: GRAP (http://www.grap.coop)
+@author: Quentin DUPONT (quentin.dupont@grap.coop)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo>
+
+    <!-- ************************************************************ -->
+    <!-- MRP Bill of Materials - Form -->
+    <!-- ************************************************************ -->
+
+    <record id="view_mrp_bom_form_produce_delay" model="ir.ui.view">
+        <field name="model">mrp.bom</field>
+        <field name="inherit_id" ref="mrp.mrp_bom_form_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='product_id']" position="after">
+                <label for="produce_delay" string="Product Manuf. Lead Time" />
```

Comment:
Same as 1st comment, i took the term of the core field "produce_delay". But I agree with you that i can change the string

Reply:
Yes in product form you need string 'Manuf' because there are many purposes but in bom view manufacture vocabulary is implicit, don't repeat yourself, KISS.